ansible_3par_docker_plugin/samples/pv_static_example.yml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: v1
+kind: PersistentVolume
+metadata:                                             
+  name: pv-first
+spec:
+    capacity:
+      storage: 10Gi
+    accessModes:
+    - ReadWriteOnce
+    flexVolume:
+      driver: hpe.com/hpe
+      options: 
+        size: "10"
```

Comment:
Do we need a size parameter here, does it impact?

Reply:
Yes,We need a size parameter in PV.